_tools/release_docs/main.go

```diff
@@ -151,6 +158,27 @@ func rewriteCodeBlocks(fileBytes []byte) []byte {
 	return []byte(strings.Join(output, "\n") + "\n")
 }
 
+var (
+	titleRE   = regexp.MustCompile(`# (.*)`)
```

Comment:
should this be anchored as ^# (.*) ?  Otherwise "### foo" will match, too.  or is that on purpose?